modules/controller/world_model_tracking.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+
+from .. import object_in_world
+
+
+class WorldModelTracking:
+    """
+    keeps track of the real world location of landing pads that
+    are to be visited or have already been vsisited
+    """
+
+    def __init__(self):
+        self.__unconfirmed_positives = []
+        self.__false_positives = []
+        self.confirmed_positive = None
+
+    def __similar(self,
+                  detection1: object_in_world.ObjectInWorld,
+                  detection2: object_in_world.ObjectInWorld) \
+                -> bool:
+        """
+        returns whether detection1 and detection2 are close enough
+        to be considered the same landing pad
+        """
+
+        DISTANCE_THRESHOLD = 2
```

Comment:
Make this a class global (i.e. move above `__init__()` ) and rename to something more descriptive (e.g. `MIN_DISTANCE_DISTINCT` ). Possibly add a comment if the variable name becomes too long.

Reply:
Actually, this can be a variable set by the constructor.